### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Max92250**
test case

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-     Movie App Documentations
+     Movie App Documentation
      Overview
```
Comment (Max92250) [inline README.md:2]: ij

**Review by Max92250 — COMMENTED** [review]
zfbfd

File: README.md (hunk 1 above)
Comment (Max92250) [reply README.md:2]: oj

**Review by Max92250 — COMMENTED** [review]
csdvd

**Review by max98703 — COMMENTED** [review]
fgrg

**Comment by Max92250** [comment]
tgtrsgh

**Pull request merged**